CHANGES.md

```diff
@@ -48,6 +48,9 @@ Unreleased
   on UNIX-like systems as a unrelated `gmake` binary might exist on Windows.
   (#3853, @kit-ty-kate)
 
+- Fix handling of aliases given on the command line (using the `@` and `@@`
+  syntax) so as to correctly handle relative paths. (#3874, @nojb)
```

Comment:
Missing the ticket number fixed: `#3850`

Reply:
Thanks, fixed.